gems/sorbet-runtime/test/types/props/_props.rb

```diff
@@ -172,7 +172,29 @@ class TestRedactedProps
     prop :secret, String, redaction: [:truncate, 4], sensitivity: []
   end
 
+  describe 'redacted props with no redaction handler' do
+    it 'raises when fetching redacted values' do
+      assert_raises do
+        d = TestRedactedProps.new
+        d.str = '12345'
+
+        # this will raise an error without a redaction handler
+        d.str_redacted
+      end
+    end
+  end
+
   describe 'redacted props' do
+    before do
+      T::Configuration.redaction_handler = lambda do |value, redaction|
+        Chalk::Tools::RedactionUtils.redact_with_directive(value, redaction)
```

Comment:
Can we take this opportunity to delete some stuff from `test_helper.rb`?

https://github.com/sorbet/sorbet/blob/6d88fd4451001b27bd0b0daa7d95e1d756330893/gems/sorbet-runtime/test/test_helper.rb#L31-L60

I'd love to just put this either in this test or into a helper function at the top of this file, rather than have it in a global `Chalk` constant.

Reply:
Good point, I'll inline it in the test.